package.json

```diff
@@ -3,7 +3,7 @@
     "version": "1.0.0",
     "description": "Testing the LAPIS API, backed by SILO, designed for easily adding new test-cases.",
     "scripts": {
-        "test": "vitest --reporter=html",
+        "test": "vitest --reporter=html --reporter=default",
```

Comment:
I think @Taepper implicitly removed the default reporter, because it might spam your terminal with diffs that are several thousand lines long.

Reply:
The spamming was an unfortunate side-effect, but not the main reason. #30 also explicitly added back this reporter, which I think is benficial